common.lic

```diff
@@ -102,7 +102,7 @@ module DRC
         return false unless kick_pile?
       when 'You really need to have at least one hand free to forage properly'
         echo 'WARNING: hands not emptied properly. Stowing...'
-        fput('stow right')
```

Comment:
This is going to explode most likely. It will stow braided vines and whatever is blocking the forage, which is bad. We only want to empty one hand.